code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -284,7 +284,7 @@
 	return 1
 
 /datum/species/proc/handle_temperature(datum/gas_mixture/breath, var/mob/living/carbon/human/H) // called by human/life, handles temperatures
-	if( (abs(310.15 - breath.temperature) > 50) && !(RESIST_HEAT in H.mutations)) // Hot air hurts :(
+	if(abs(310.15 - breath.temperature) > 50) // Hot air hurts :(
```

Comment:
I'd...
Wow, actually, I just realized I totally have a double standard. This is fine. I like how cold resist requires no breathing to go into space. Keeping in line, this shouldn't affect breath, I suppose.